site/cds_rdm/legacy/redirector.py

```diff
@@ -58,9 +58,18 @@ def legacy_files_redirect(legacy_id, filename):
         record = get_record_by_version(parent_pid.pid_value, version)
     except PermissionDeniedError:
         return abort(403)
+    filename_ext = filename.split(".")[-1]
     # Directly download files from redirected link to replicate the `allfiles-` behaviour from legacy
     if filename.startswith("allfiles-"):
         url_path = record["links"]["archive"]
+    # If the file is not previewable, redirect to the file download link instead
+    elif not current_app.config["IIIF_FORMATS"].get(filename_ext, None):
+        url_path = url_for(
```

Comment:
we dont have this link in the record links?

Reply:
We have the content link
```
/api/records/<record_id>/files/<filename>/content
```
We can re-use that, or is a non-api link prefered?